src/django/api/extended_fields.py

```diff
@@ -70,11 +109,19 @@ def create_extendedfields_for_listitem(item, fields, values):
     if item.id is None:
         return False
     contributor = item.source.contributor
+
     for field in RAW_DATA_FIELDS:
         if field in fields:
             field_value = values[fields.index(field)]
             create_extendedfield(field, field_value, item, contributor)
 
+            # Add a facility_type extended field if the user only
+            # submitted a processing_type
+            if ((field == 'processing_type') and
```

Comment:
I think we may want to do this in reverse as well since each facility type is also included as a processing type. Better to err on the side of having the facility show up when searching either field for a particular value.

```python
            if ((field == 'facility_type') and
               'processing_type' not in fields):
                create_extendedfield('processing_type', field_value, item,
                                     contributor)
```